src/logging.js

```diff
@@ -45,9 +45,17 @@ function handle (level, args) {
   if (!log) {
     console.error("Log not initialised"); // eslint-disable-line no-console
   }
-  // Take the first arg as context
-  // apply util.format to the second arg, with the remaining args as input
-  const message = args[0] + " " + util.format.apply(null, args.slice(1));
+
+  // If the first arg does NOT contain '%', assume it is context and append it onto the
```

Comment:
If my simple, you mean a string with no '%s's, the context will get set to that and then `args = []` and `util.format` will return an empty string. So the logged message is still correct.